types/component/ChatList/ChannelImage.types.ts

```diff
@@ -2,18 +2,19 @@
 
 import {ImageStyle} from 'react-native-fast-image';
 import {StyleProp} from 'react-native';
+import {BetterSocialChannelType} from '../../database/schema/ChannelList.types';
 
 export type ChannelImageProps = React.FC & {
   Big: React.FC<ChannelImageMainProps>;
   Small: React.FC<ChannelImageBadgeProps>;
 };
 
 export type ChannelImageMainProps = {
-  type: 'COMMUNITY' | 'GROUP' | 'GROUP_INFO';
+  type: BetterSocialChannelType;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing the hardcoded string literals with the `BetterSocialChannelType` enum in the `type` field of `ChannelImageMainProps` is a good practice. It ensures that only valid types can be used, reducing the risk of errors.



<!-- This is an auto-generated comment by OSS CodeRabbit -->